src/debugger/appWorker.ts

```diff
@@ -200,7 +200,13 @@ export class MultipleLifetimesAppWorker {
     }
 
     public start(warnOnFailure: boolean = false): Q.Promise<any> {
-        return this.createSocketToApp(warnOnFailure);
+        return Packager.isPackagerRunning(Packager.getHostForPort(this.packagerPort))
+            .then(running => {
+                if (running) {
+                    return this.createSocketToApp(warnOnFailure);
+                }
+                throw new RangeError(`Cannot attach to packager. Are you sure there is a packager and it is running in the port ${this.packagerPort}? If your packager is configured to run in another port make sure to add that to the setting.json.`);
```

Comment:
No specific reason, I saw it used on other places and followed it. Should I change it to `Error`